index.bs

```diff
@@ -3586,8 +3561,7 @@ invoked, must run these steps:
 
 <div class=algorithm>
```

Comment:
Preexisting issue: `<div class=algorithm>` is, I believe, not processed by Bikeshed in the same ways as `<div algorithm>`. In particular you're not getting the nice user-facing feature where you can click on a `<var>` and all the instances of it are highlighted, or the spec-developer facing feature of checking for variables that only appear once (usually due to typos).

Reply:
Ah, interesting - thanks! I'll take this on separately. Generated a lot of errors for variables used only once seems to be a problem, likely in normative blocks outside algorithms that should be non-normative. I'll need to work through them.